mlir/tools/mlir-miopen-lib/mlir-miopen-lib.h

```diff
@@ -0,0 +1,85 @@
+//===-------------- mlir-miopen-lib.h - C API ---------------------*- C -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM
+// Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef MLIR_C_MIIR_LIBMIIR_H
+#define MLIR_C_MIIR_LIBMIIR_H
+
+#include <stddef.h>
+
+#define MIIR_VERSION_FLAT 0
+
+enum MiirStatus {
+  MIIR_SUCCESS = 0,
+  MIIR_INVALID_PARAM,
+  MIIR_INVALID_MODULE,
+  MIIR_BUILD_FAILURE
+};
+typedef enum MiirStatus MiirStatus;
+
+/*! @brief The MLIR handle used for lowering and code generation
+ */
+typedef void *MiirHandle;
```

Comment:
typo

Reply:
@atamazov I updated the naming convention: mlirmiopen -> miir